### PR TITLE
docs: update jcloud gpu usage

### PR DESCRIPTION
**Pull request by ZiniuYu**
Add instructions on using GPU in Jcloud

### CODE REVIEW

File: docs/hosting/on-jcloud.md [hunk 1]
```diff
@@ -28,9 +24,14 @@ executors:
...
+```
 
-Executors now must start with `jinahub+docker://` as it is required by JCloud. We currently provide containerized executors [`jinahub+docker://CLIPTorchEncoder`](https://hub.jina.ai/executor/gzpbl8jh) and [`jinahub+docker://CLIPOnnxEncoder`](https://hub.jina.ai/executor/2a7auwg2) on Jina Hub. They are automatically synced on the new release of `clip_server` module. 
+Executors must start with `jinahub+docker://` as it is required by JCloud. We currently provide containerized executors [`jinahub+docker://CLIPTorchEncoder`](https://hub.jina.ai/executor/gzpbl8jh) and [`jinahub+docker://CLIPOnnxEncoder`](https://hub.jina.ai/executor/2a7auwg2) on Jina Hub. They are automatically synced on the new release of `clip_server` module. 
+Please refer [here](https://docs.jina.ai/fundamentals/jcloud/yaml-spec/#gpu) for more details on using GPU in JCloud.
+Notice that you must specify the correct docker image tag for your executor to utilize the GPU. For example `jinahub+docker://CLIPTorchEncoder/0.5.2-gpu`. 
+See the 'Tag' section in [CLIPTorchEncoder](https://hub.jina.ai/executor/gzpbl8jh) and [CLIPOnnxEncoder](https://hub.jina.ai/executor/2a7auwg2) for GPU docker image tags.
```
Comment (numb3r3) [inline docs/hosting/on-jcloud.md:34]: It's better to provide two YAML examples to show how to use GPU in jcloud

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/hosting/on-jcloud.md (hunk 1 above)
Comment (ZiniuYu) [reply docs/hosting/on-jcloud.md:34]: which is preferred? shared or dedicated? https://docs.jina.ai/fundamentals/jcloud/yaml-spec/#gpu

Comment (numb3r3) [reply docs/hosting/on-jcloud.md:34]: Leave this question to the user. In our example, shared is fine.

File: docs/hosting/on-jcloud.md [hunk 2]
```diff
@@ -28,9 +24,14 @@ executors:
 
 ````
 
-Note that, `port` is unnecessary here as JCloud will assign a new URL for any deployed service. 
+```{tip}
+`port` is unnecessary here as JCloud will assign a new URL for any deployed service.
```
Comment (numb3r3) [inline docs/hosting/on-jcloud.md:28]:
```suggestion
`port` is unnecessary here as JCloud will assign a new hostname and port for any deployed service. 
```

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/hosting/on-jcloud.md [hunk 3]
```diff
@@ -28,9 +24,26 @@ executors:
...
+```
+
+Executors must start with `jinahub+docker://` as it is required by JCloud. We currently provide containerized executors [`jinahub+docker://CLIPTorchEncoder`](https://hub.jina.ai/executor/gzpbl8jh) and [`jinahub+docker://CLIPOnnxEncoder`](https://hub.jina.ai/executor/2a7auwg2) on Jina Hub. They are automatically synced on the new release of `clip_server` module. 
+
+To enable GPU on JCloud, you need to configure it in the YAML file and use prebuilt docker GPU images. For example,
+
+```yaml
+jtype: Flow
+executors:
+  - uses: jinahub+docker://CLIPTorchEncoder/0.5.2-gpu
```
Comment (numb3r3) [inline docs/hosting/on-jcloud.md:38]:
```suggestion
  - uses: jinahub+docker://CLIPTorchEncoder/latest-gpu
```

File: docs/hosting/on-jcloud.md [hunk 4]
```diff
@@ -28,9 +24,26 @@ executors:
...
+    jcloud:
+      resources:
+        gpu: shared
+```
 
-Executors now must start with `jinahub+docker://` as it is required by JCloud. We currently provide containerized executors [`jinahub+docker://CLIPTorchEncoder`](https://hub.jina.ai/executor/gzpbl8jh) and [`jinahub+docker://CLIPOnnxEncoder`](https://hub.jina.ai/executor/2a7auwg2) on Jina Hub. They are automatically synced on the new release of `clip_server` module. 
+Please refer [here](https://docs.jina.ai/fundamentals/jcloud/yaml-spec/#gpu) for more details on using GPU in JCloud.
+Notice that you must specify a docker image GPU tag for your executor to utilize the GPU. For example `0.5.2-gpu`.
```
Comment (numb3r3) [inline docs/hosting/on-jcloud.md:45]: change the tag

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by numb3r3 — APPROVED** [review]

**Pull request merged**